src/features/events/hooks/useCreateEvent.ts

```diff
@@ -20,6 +21,10 @@ export default function useCreateEvent(orgId: number): useCreateEventReturn {
       eventBody
     );
     dispatch(eventCreated(event));
+    env.router.push(
+      `/organize/${orgId}/projects/${event.campaign?.id}/events/${event.id}`
```

Comment:
I think we should always use the `getEventUrl()` utility to generate event URLs.

Reply:
okay!